src/ducks/upload/styles.styl

```diff
@@ -0,0 +1,187 @@
+@import 'cozy-ui'
+@import '../../styles/mimetypes'
+
+.upload-queue
+    @extend $popover
+    position fixed
+    bottom   em(8px)
+    right    em(24px)
+    height   em(210px)
+    width    em(480px)
+    max-width 90%
+    border   1px solid grey-09
```

Comment:
I know you probably copied and pasted but could you change the greys variables with the new ones? that will do less changes in the future